aspnet/fundamentals/configuration.rst

```diff
@@ -22,7 +22,6 @@ To work with settings in your ASP.NET application, it is recommended that you on
 At its simplest, the ``Configuration`` class is just a collection of ``Providers``, which provide the ability to read and write name/value pairs. You must configure at least one provider in order for ``Configuration`` to function correctly. The following sample shows how to test working with ``Configuration`` as a key/value store:
```

Comment:
We added the notion of Sources which is now what you add to Configuration (the providers are built from sources, not sure if that distinction is worth calling out)